src/libratbag-enums.h

```diff
@@ -327,3 +327,21 @@ enum ratbag_macro_event_type {
 	RATBAG_MACRO_EVENT_KEY_RELEASED,
 	RATBAG_MACRO_EVENT_WAIT,
 };
+
+/**
+ * @ingroup enums
+ *
+ * Device Types
+ * Describes the types specified in the .device files
+ */
+enum ratbag_device_type {
+	// Used when no DeviceType property could be found
+	TYPE_UNSPECIFIED = 0,
+	// Used for mice
+	TYPE_MOUSE,
+	// Used for keyboards
+	TYPE_KEYBOARD,
+	// Used for any other device type than a mouse or a keyboard
+	// for example headsets, mousepads etc.
+	TYPE_OTHER
```

Comment:
nitpick: I would move this up, to the 2nd position. The argument being it is a "special" type and when we add new types we will have a special type in the middle of the list.

Also adjust the documentation after.

Reply:
realistically, mice are the only one needing a special case, because the first two buttons shouldn't be rempped. I don't think that there will be many special cases in the future.

Reply:
Well yes, but I would still like to keep the enum in order. Should be a pretty easy fix.

Reply:
I think it is a good idea as well

Reply:
its swapped now